### ISSUE TITLE
Test files using the Brotli compression feature would be helpful

### ISSUE THREAD

**Issue by boardhead** [c2pa-org/public-testfiles#6]
I can't find any test files that use the Brotli compression feature.  Samples of both a compressed manifest

https://c2pa.org/specifications/specifications/1.4/specs/C2PA_Specification.html#_compressed_manifests

and a compressed box

https://c2pa.org/specifications/specifications/1.4/specs/C2PA_Specification.html#_compressed_boxes

would be helpful.

- Phil

**Comment by lrosenthol** [comment]
@boardhead Yes it would.  I am not aware of any creation implementations supporting the new compressed manifests feature - but we hope to have some soon!
